superset-frontend/cypress-base/cypress/integration/explore/visualizations/histogram.js

```diff
@@ -72,7 +72,6 @@ export default () =>
             comparator: 'boy',
             clause: 'WHERE',
             sqlExpression: null,
-            fromFormData: true,
```

Comment:
what is this field for?

Reply:
This was used to detect whether an adhoc metric/filter comes from stored metadata. It is used to automatically open the edit popover when adding a new metric/filter.